flink-ai-flow/lib/airflow/airflow/contrib/jobs/periodic_manager.py

```diff
@@ -42,48 +48,60 @@ def shutdown(self):
+                                       minute=cron_items[1],
+                                       hour=cron_items[2],
+                                       day=cron_items[3],
+                                       month=cron_items[4],
+                                       day_of_week=cron_items[5],
+                                       year=cron_items[6])
+                elif len(cron_items) == 6:
+                    return CronTrigger(second=cron_items[0],
+                                       minute=cron_items[1],
+                                       hour=cron_items[2],
+                                       day=cron_items[3],
+                                       month=cron_items[4],
+                                       day_of_week=cron_items[5])
+                else:
+                    raise ValueError('The cron expression {} is incorrect format, follow the pattern: '
+                                     'second minute hour day month day_of_week optional(year).'.format(cron_items))
+
             self.sc.add_job(id=self._generate_job_id(run_id, task_id),
                             func=trigger_periodic_task, args=(self.mailbox, run_id, task_id),
-                            trigger=CronTrigger.from_crontab(periodic_config['cron']))
+                            trigger=build_cron_trigger(periodic_config['cron']))
         elif 'interval' in periodic_config:
-            interval_config: dict = periodic_config['interval']
-            if 'seconds' in interval_config:
-                seconds = interval_config['seconds']
-            else:
-                seconds = 0
-            
-            if 'minutes' in interval_config:
-                minutes = interval_config['minutes']
-            else:
-                minutes = 0
-            
-            if 'hours' in interval_config:
-                hours = interval_config['hours']
-            else:
-                hours = 0
-                
-            if 'days' in interval_config:
-                days = interval_config['days']
-            else:
-                days = 0
-            
-            if 'weeks' in interval_config:
-                weeks = interval_config['weeks']
-            else:
-                weeks = 0
-            
-            if seconds < 10 and 0 >= minutes and 0 >= hours and 0 >= days and 0 >= weeks:
-                self.log.error('Interval mast greater than 20 seconds')
-                return 
+            interval_expr: str = periodic_config['interval']
+            interval_items = interval_expr.split(',')
+            if len(interval_items) != 5:
+                raise ValueError('The interval expression {} is incorrect format, follow the pattern: '
+                                 'weeks,days,hours,minutes,seconds.'.format(interval_items))
+            temp_list = []
+            is_zero = True
+            for item in interval_items:
+                if item is None or '' == item.strip():
```

Comment:
I suppose None or '' are illegal interval and will raise exception by strip(). We don't need to parse them as 0.